### DISCUSSION TITLE
Battery power behavior with Deye inverter

### DISCUSSION THREAD

**Discussion by sofkaski** [davidrapan/ha-solarman#116] [General]
Has anybody had a similar observation:

I have had recently my battery disconnected for maintenance.  The sensor `sensor.inverter_battery_power` shows still some varying positive power below 10 W. This must be some phantom since the battery circuit is physically open. (My setup is using Deye LV 3-Phase Hybrid Inverter. Inverter definition file is deye_sg04lp3.yaml.).

There is nothing wrong in the integration since the value comes from the inverter. I was just wondering if anybody has some idea why that happens?

**Comment by davidrapan** [comment]
And did that value change during that disconnected period or stayed frozen?

**Comment by sofkaski** [comment]
It is varying. It seems that the variance correlates to overall power consumption or production, or both. Example:

![image](https://github.com/user-attachments/assets/1bc82fe4-6ec8-4771-839b-cb2a8034e58c)

**Reply by davidrapan** [reply]
That's weird but my guess is that has something to do with the precision again. You are gonna have to ask Deye about that. 🤔
